python/ola/PidStore.py

```diff
@@ -157,14 +157,58 @@ def ValidateAddressing(self, args, command_class):
         return False
     return True
 
-  def __cmp__(self, other):
-    return cmp(self._value, other._value)
+  def _GroupCmp(self, a, b):
+    def setToDescription(x):
+      def descOrNone(y):
+        return y.GetDescription() if y is not None else "None"
+      return ':'.join([descOrNone(x[f])
+                       for f in [RDM_GET, RDM_SET, RDM_DISCOVERY]])
+    ax = setToDescription(a)
+    bx = setToDescription(b)
+    return (ax > bx) - (ax < bx)  # canonical for cmp(a,b)
+
+  def __eq__(self, other):
+    if not isinstance(other, self.__class__):
+      return False
+    return (self.value == other.value and
+            self.name == other.name and
```

Comment:
As below, I think name is probably irrelevant. I guess requests/responses might be for some merge scenario.

Reply:
Done.